### ISSUE TITLE
A11Y_VSCode_ClearSearchResult_ScreenReader:Screen reader doesn't announce anything after clicking on the 'clear search result' button

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode#76862]
**Environment Details:**
VSCode Version : 1.36.1 (user setup)
OS Version : MacOS Mojave 10.14.5

**Additional Details:**
MAS Violated : MAS 1.3.1
Screen reader Versions : Voice Over

**Repro Steps:**
1)Open Visual studio code editor    
2)Using keyboard navigation View Menu Bar
3)Select search option
4)Make sure it's possible to clear results
6.Start screen reader

**Actual:**
 Screen reader does not announce anything after clicking on the 'clear search result' button

This issue is observed in Mac + Voice over

**Expected:**
Screen reader should announce that the search results are cleared after clicking on the 'clear search result' button

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader user will not get the exact information.

**MAS Reference:**
**MAS 1.3.1 -** https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11Y_VSCode_ClearSearchResult_ScreenReader.pptx](https://github.com/microsoft/vscode/files/3367838/A11Y_VSCode_ClearSearchResult_ScreenReader.pptx)
